src/js/view/controls/controlbar.js

```diff
@@ -202,13 +204,15 @@ export default class Controlbar {
 
         const nextUpTip = SimpleTooltip(elements.next.element(), 'next', localization.nextUp, () => {
             const nextUp = _model.get('nextUp');
```

Comment:
When model 'nextUp' changes, 'feedShownId' should be reset. (since this is just a tooltop, and the 'nextShown' is only handled once you could just generate this id once there. Use the change listener on line [283](https://github.com/jwplayer/jwplayer/pull/3010/files#diff-da6bb0847c43fc9abf65d5ce7b9829c5R283)
```js
_model.change('nextUp', (model, nextUp) => {
   let tipText = localization.nextUp;
```

Reply:
But in this case wouldn't the feedShownId would be the same until the nextUp item changes? This is contrary to the AC (An identifier is generated with each new feed_shown event) which I interpret as meaning a different id is generated every time the UI appears.